.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+Please include a summary of the change. Please also include relevant
+motivation and context. See
+[contribution guidelines](https://github.com/oneapi-src/oneMKL/blob/master/CONTRIBUTING.md)
+for more details. If the change fixes an issue not documented in the project's
+Github issue tracker, please document all steps necessary to reproduce it.
+
+Fixes # (github issue)
+
+# Checklist
+
+## All Submissions
+
+- [ ] Do all unit tests pass locally? Attach a log.
+- [ ] Have you formatted the code using clang-format?
+
+## New interfaces
+
+- [ ] Have you provided motivation for adding a new feature as part of RFC and it was accepted? # (RFC)
+- [ ] What version of oneAPI spec the interface is targeted?
+- [ ] Complete [New features](pull_request_template.md#new-features) checklist
```

Comment:
This link comes to the same page. Did you mean feature_request page?

Reply:
No, the link is correct. The point is that interfaces are also feature, so Features' checklist should be completed as well.